noxfiles/ci_nox.py

```diff
@@ -357,3 +221,32 @@ def python_build(session: nox.Session, dist: str) -> None:
         dist,
         external=True,
     )
+
+
+@nox.session()
+def upload_coverage(session: nox.Session) -> None:
+    """
+    Format, combine and upload the test coverage files.
```

Comment:
Should we add a note here that this won't work locally because of the need for the upload key? I think it's fine for it to not work locally because we wouldn't want testing locally to modify our baseline test number, it would just make it clear in the help output.

```suggestion
    Format, combine and upload the test coverage files. Note that this command only works in CI.
```